src/components/Payment/Payment.tsx

```diff
@@ -96,7 +98,7 @@ const Payment = ({
                   <small>/{t(`account:periods.${activeSubscription.period}`)}</small>
```

Comment:
Can you add a transtlation for `account:periods.granted`? Maybe just 'once'?

Reply:
You mean like this? 
<img width="800" alt="image" src="https://user-images.githubusercontent.com/37805865/217209892-2c915ee0-eb98-4fe8-b7b3-3c538013762f.png">

Reply:
Yep!